attribute/generic.go

```diff
@@ -0,0 +1,25 @@
+package attribute
+
+import (
+	"github.com/getyoti/yoti-go-sdk/anchor"
+)
+
+//Generic is a Yoti attribute which returns a generic value
+type Generic struct {
+	Name    string
+	Value   string
```

Comment:
Do you really want all your attributes to have a string representation? This may get difficult for non-string values... ie. a selfie pic.

Reply:
This Generic type is only returned for the GetAttribute function. We are ensuring we make each of the SDKs work in the same way as much as possible, using the Java SDK as a base.
 
Although we considered only having a byte value, the Java SDK returns a string value, and so we are making this work in the same way, as changing it in Java would require a breaking change. 

It might produce strange results for image types, but it also gives a convenience method for what most of the attributes are - string types.

Reply:
I think the "make each of the SDKs work in the same way as much as possible" could make things really difficult if you try too hard... I think this is a relative statement... relative to the language constraints...

Reply:
Yes, it's a question of how far you take "as much as possible". @bucky-boy - we're sticking with string representations, right?